cypress/e2e/checkout.cy.js

```diff
@@ -1,11 +1,45 @@
 /// <reference types='cypress' />
+import { HomePage } from "../support/homePage.pageObject";
+import { Cart } from "../support/cart.pageObject";
 
-describe('', () => {
+describe('demoblaze', () => {
+  let user;
+  const homePage = new HomePage();
+  const cartPage = new Cart();
   before(() => {
+    cy.task('generateUser').then(generateUser => {
+      user = generateUser;
+    });
 
+    homePage.visit();
   });
 
-  it('', () => {
+  it('should be able to add and delete producr from the cart', () => {
+    homePage.categoryList('Laptops').click();
```

Comment:
you can create the method: click on the button/category/etc

Reply:
still not changed